packages/framework/src/Controller/Admin/ParameterValueController.php

```diff
@@ -104,4 +108,49 @@ public function editAction(Request $request, int $id): Response
             ],
         );
     }
+
+    /**
+     * @param \Symfony\Component\HttpFoundation\Request $request
+     * @param int $id
+     * @return \Symfony\Component\HttpFoundation\Response
+     */
+    #[Route(path: '/parameter-values/edit/{id}', name: 'admin_parametervalues_edit', requirements: ['id' => '\d+'])]
+    public function parameterValuesEditAction(Request $request, int $id): Response
+    {
+        $parameter = $this->parameterFacade->getById($id);
+        $parameterValues = $this->parameterFacade->getParameterValuesByParameter($parameter);
+        $parameterValuesConversionDataIndexedByParameterValueId = $this->parameterValueConversionDataFactory->createForNumericConversion($parameterValues);
+
+        $form = $this->createForm(SliderParameterValuesUpdateFormType::class, $parameterValuesConversionDataIndexedByParameterValueId, [
+            'data' => $parameterValuesConversionDataIndexedByParameterValueId,
+        ]);
+        $form->handleRequest($request);
+
+        if ($form->isSubmitted() && $form->isValid()) {
+            $this->parameterFacade->updateParameterValuesByConversion($parameter, $parameterValuesConversionDataIndexedByParameterValueId);
+            $this->addSuccessFlashTwig(
+                t('Parameter values associated with parameter - <strong><a href="{{ url }}">{{ parameter.name }}</a></strong> modified.'),
```

Comment:
the usage of `{{ parameter.name }}` seems quite unusual. Why do not you use just `{{ parameterName }}` variable?